src/manager.cpp

```diff
@@ -0,0 +1,12 @@
+#include "manager.hpp"
+
+namespace vpd
+{
+Manager::Manager(std::shared_ptr<boost::asio::io_context>& ioCon,
```

Comment:
I wanted shared_ptr to be taken by value as the object deletion will be taken care from with shared ownership. But since these are happening within the class scope, as you said we don't need shared ownership. so am ok with having const& for shared_ptr.